lib/chef/provisioning/transport/winrm.rb

```diff
@@ -22,27 +22,24 @@ class WinRM < Chef::Provisioning::Transport
       # The actual connection is made as ::WinRM::WinRMWebService.new(endpoint, type, options)
       #
       def initialize(endpoint, type, options, global_config)
-        @endpoint = endpoint
-        @type = type
         @options = options
+        @options[:endpoint] = endpoint
+        @options[:transport] = type
+
+        # WinRM v2 switched from :pass to :password
+        # we accept either to avoid having to update every driver
+        @options[:password] = @options[:password] || @options[:pass]
+        
         @config = global_config
       end
 
-      attr_reader :endpoint
-      attr_reader :type
       attr_reader :options
       attr_reader :config
 
       def execute(command, execute_options = {})
         output = with_execute_timeout(execute_options) do
-          session.set_timeout(execute_timeout(execute_options))
-          command_executor = ::WinRM::CommandExecutor.new(session)
           block = Proc.new { |stdout, stderr| stream_chunk(execute_options, stdout, stderr) }
-          if execute_options[:raw]
-            command_executor.run_cmd(command, &block)
-          else
-            command_executor.run_powershell_script(command, &block)
-          end
+          session.run(command, &block)
```

Comment:
@mwrock @tyler-ball we need to default to session.run_powershell_script and only use session.run_cmd if raw options are passed the ```:raw``` options
/cc @ajeba99